app/src/main/java/nextstep/signup/MainActivity.kt

```diff
@@ -22,33 +36,120 @@ class MainActivity : ComponentActivity() {
         super.onCreate(savedInstanceState)
         setContent {
             SignupTheme {
-
+                Scaffold(
+                    topBar = {
+                        Row(
+                            modifier = Modifier
+                                .fillMaxWidth()
+                                .padding(top = 62.dp),
+                            horizontalArrangement = Arrangement.Center
+                        ) {
+                            TitleView("Welcome to Compose \uD83D\uDE80")
+                        }
+                    },
+                    content = { paddingValues ->
+                        Column(
+                            modifier = Modifier
+                                .fillMaxSize()
+                                .padding(paddingValues),
+                            horizontalAlignment = Alignment.CenterHorizontally
+                        ) {
+                            TextFieldView(
+                                Modifier.padding(16.dp),
+                                "Username",
+                                KeyboardType.Text
+                            )
+                            TextFieldView(
+                                Modifier.padding(16.dp),
+                                "Email",
+                                KeyboardType.Email
+                            )
+                            TextFieldView(
+                                Modifier.padding(16.dp),
+                                "Password",
+                                KeyboardType.Password
+                            )
+                            TextFieldView(
+                                Modifier.padding(16.dp),
+                                "Password Confirm",
+                                KeyboardType.Password
+                            )
+                            SingUpButtonView(
+                                Modifier.padding(16.dp)
+                            )
+                        }
+                    },
+                    modifier = Modifier.padding(horizontal = 33.dp)
+                )
             }
         }
     }
 }
 
+@Preview
 @Composable
-fun HelloView(
-    name: String,
-    modifier: Modifier = Modifier) {
-    Text(text = "$name 안녕하세요!",
-        fontSize = 20.sp,
-        color = Color.White,
+private fun SingUpButtonView(
+    modifier: Modifier = Modifier
+) {
+    Button(
+        onClick = { /*TODO*/ },
+        colors = ButtonDefaults.buttonColors(containerColor = Color(0xFF2196F3)),
         modifier = modifier
-            .background(
-                brush = Brush.linearGradient(
-                    colors = listOf(Color.Red, Color.Green),
-                    start = Offset.Zero,
-                    end = Offset.Infinite,
-                ),
-                shape = RoundedCornerShape(16.dp)
+            .fillMaxWidth()
+    )
+    {
+        Text(text = "Sign Up", fontSize = 14.sp, color = Color(0xFFFFFFFF))
+    }
+}
```

Comment:
컴포즈는 C 스타일의 중괄호 줄바꿈보다는, Java Kotlin 스타일의 중괄호 줄바꿈을 해주시면 되어요!
```suggestion
    ) {
        Text(text = "Sign Up", fontSize = 14.sp, color = Color(0xFFFFFFFF))
    }
}
```

Reply:
아~ 감사합니다 : 😊